packages/styles/stories/Components/search-field/search-field.stories.js

```diff
@@ -0,0 +1,31 @@
+import defaultExampleHtml from "./default.example.html?raw";
+import '../../../src/icon.scss';
+import '../../../src/popover.scss';
+import '../../../src/list.scss';
+import '../../../src/form-label.scss';
+import '../../../src/form-message.scss';
+import '../../../src/select.scss';
+import '../../../src/dialog.scss';
+import '../../../src/toolbar.scss';
+import '../../../src/button.scss';
+import '../../../src/icon.scss';
+import '../../../src/layout-grid.scss';
+import '../../../src/form-item.scss';
+import '../../../src/search-field.scss';
+
+export default {
+  title: 'Components/Search Field',
+  parameters: {
+    description: ``,
+    tags: ['btp']
+  }
+};
+export const Default = () => defaultExampleHtml;
+Default.parameters = {
+  docs: {
+    description: {
+      story: `
+        `
+    }
+  }
+};
```

Comment:
What's up with these empty strings for `story` and `description`? If you do not need them just remove